src/data-table/_data-table.scss

```diff
@@ -24,6 +24,7 @@
   white-space: nowrap;
   font-size: 13px;
   background-color: unquote("rgb(#{$color-white})");
+  -webkit-transform: translate3d(0,0,0);
```

Comment:
@Garbee @sgomes Would something like this, be a decent medium?

`/* Prefixed to target Safari specific bug */`
`-webkit-transform: translate3d(0,0,0);`

I'm not familiar with how you guys want in-line commenting done, so maybe you just want to update this yourself? Will submit a new pull request, no problem, but let me know either way - I know this is all reasonably trivial at this point.